packages/vscode-extension/src/network/components/Tabs/HighlightedCodeBlock.tsx

```diff
@@ -77,6 +78,7 @@ const HighlightedCodeBlock = ({
       language={language}
       showLanguage={false}
       addDefaultStyles={false}
+      delay={HIGHLIGHT_THROTTLE}
```

Comment:
Do I understand correctly, that this helps when quickly switching between code entries, to prevent highlighter from running until the entry is open for some time?